src/cli/Extract.cpp

```diff
@@ -47,10 +48,6 @@ int main(int argc, char **argv)
         return 1;
```

Comment:
Maybe use EXIT_FAILURE (and EXIT_SUCCESS) from <cstdlib> instead of hard-coded 1 and 0?